install/kubernetes/templates/operator_clusterrole.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if and .Values.serviceAccount.create .Values.podWatcher.enabled }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{.Chart.Name}}-operator
+  labels:
+  {{- include "tetragon-operator.labels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - cilium.io
+    resources:
+      - tetragonpods
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - cilium.io
+    resources:
+      - tetragonpods/finalizers
+    verbs:
+      - update
+  - apiGroups:
+      - cilium.io
+    resources:
+      - tetragonpods/status
+    verbs:
+      - get
+      - patch
+      - update
```

Comment:
```suggestion
```
let's delete these. the operator has full access to tetragonpods resources anyways.

Reply:
sure